### PR TITLE
Allowing no test

### PR DESCRIPTION
**Pull request by kegl**

### CODE REVIEW

**Comment by kegl** [comment]
We simply allow `get_test_data()` in `problem.py` to return `None, None` in case there is no test file. It could be useful even in the autism challenge if they want to reproduce the backend public leaderboard.

File: rampwf/tests/kits/titanic_no_test/titanic_no_test_starting_kit.ipynb [hunk 1]
```diff
@@ -0,0 +1,1337 @@
+{
```
Comment (jorisvandenbossche) [inline rampwf/tests/kits/titanic_no_test/titanic_no_test_starting_kit.ipynb]: no need to add the notebook here I think

Comment (kegl) [reply rampwf/tests/kits/titanic_no_test/titanic_no_test_starting_kit.ipynb]: True, but doesn't the test need it?

**Comment by aboucaud** [comment]
@kegl please fix flake8
```
rampwf/tests/test_kits.py:61:1: E303 too many blank lines (3)
```

**Pull request merged**